src/prepack-cli.js

```diff
@@ -129,6 +135,12 @@ function run(Object, Array, console, JSON, process, prepackStdin, prepackFileSyn
         // their errors to the console, but exit with an error code.
         process.exit(1);
       }
+      if (x instanceof DeferredErrorsError) {
+        // Report the general failure reason. The specific errors were already
+        // handled and reported by the error-handler
+        console.log(x.message);
```

Comment:
It is somewhat unusual for a compiler to report "No object code was generated because there were compilation errors" and this seems sufficiently analogous that I wonder why this is necessary.